### PR TITLE
Deepika

### PR DESCRIPTION
**Pull request by DeepikaTamma**
Made changes by merging mgmt server.

### CODE REVIEW

**Comment by DeepikaTamma** [comment]
done with pull request

File: src/main/java/com/example/demo/controller/BiAnalyticCamDefController.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
+package com.example.demo.controller;
```
Comment (SriDatta123) [inline src/main/java/com/example/demo/controller/BiAnalyticCamDefController.java:1]: /analytics is repeated 3 times

File: src/main/java/com/example/demo/controller/CustomerController.java [hunk 2]
```diff
@@ -19,17 +37,148 @@
...
+	
+	@Autowired
+	CustomersRepo customerRepo;
+	
+	@GetMapping(value = "/customer/listCustomers_1_0")
 	public Object listCustomers_1_0()
 	{
 //		throw new ConstraintViolationException("", null);
 		return customerService.listCustomers_1_0();
 	}
```
Comment (SriDatta123) [inline src/main/java/com/example/demo/controller/CustomerController.java:56]: Please add listCustomers_1_0 back

File: src/main/java/com/example/demo/entity/BiReportSubcatPriority.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package com.example.demo.entity;
+
+public enum BiReportSubcatPriority {
+	H,
+	L,
+	VL,
+	O
```
Comment (SriDatta123) [inline src/main/java/com/example/demo/entity/BiReportSubcatPriority.java:7]: Where is this class used?
Please comment inside if you can

File: src/main/java/com/example/demo/entity/BiReportSubcatServiceType.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package com.example.demo.entity;
+
+public enum BiReportSubcatServiceType {
```
Comment (SriDatta123) [inline src/main/java/com/example/demo/entity/BiReportSubcatServiceType.java:3]: Where is this class used?
Please comment inside if you can

File: src/main/java/com/example/demo/entity/BiReportSubcatStatus.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
+package com.example.demo.entity;
```
Comment (SriDatta123) [inline src/main/java/com/example/demo/entity/BiReportSubcatStatus.java:1]: Where is this class used?
Please comment inside if you can

File: src/main/java/com/example/demo/entity/ResponseEntity.java [hunk 6]
```diff
@@ -0,0 +1,8 @@
+package com.example.demo.entity;
```
Comment (SriDatta123) [inline src/main/java/com/example/demo/entity/ResponseEntity.java:1]: Is this empty class needed?

File: src/main/java/com/example/demo/view/View.java [hunk 7]
```diff
@@ -0,0 +1,29 @@
+package com.example.demo.view;
```
Comment (SriDatta123) [inline src/main/java/com/example/demo/view/View.java:1]: Where is this class used?

File: src/main/resources/META-INF/additional-spring-configuration-metadata.json [hunk 8]
```diff
@@ -0,0 +1,52 @@
+{"properties": [
```
Comment (SriDatta123) [inline src/main/resources/META-INF/additional-spring-configuration-metadata.json:1]: where is this json used?

**Review by SriDatta123 — CHANGES_REQUESTED** [review]
Please make necessary changes and remove empty class files

**Comment by SriDatta123** [comment]
Please create new pull request after completing above

**Comment by SriDatta123** [comment]
Create address controller and seperate requests starting with /address and /customer

**Pull request closed without merging**